docs/async_mode.adoc

```diff
@@ -0,0 +1,11 @@
+= Async Mode
+:toc:
+:toc-title: Contents
+:toclevels: 1
+
+== Chronicle Async Mode
+
+Chronicle Async Mode provides a high performance, multi-writer, multi-reader, buffered queue implementation
+which can be used in your application code.
+
+For more information on Chronicle Async Mode please visit the https://portal.chronicle.software/[Chronicle User Portal].
```

Comment:
should link to https://portal.chronicle.software/docs/queue/

Reply:
will fix - thanks